### PR TITLE
Docs 11996 what not imported bsj

### PR DESCRIPTION
**Pull request by bjoson-mulesoft**

### CODE REVIEW

**Review by fedebrasburg — DISMISSED** [review]

**Review by fedebrasburg — DISMISSED** [review]

**Review by nledesma — COMMENTED** [review]
LGTM! 🚀 Lets wait for @fedebrasburg  approval, who already gave feedback

EDIT: just approved it seconds ago, hehe

File: modules/ROOT/pages/import-api-task.adoc [hunk 1]
```diff
@@ -3,13 +3,13 @@ ifndef::env-site,env-github[]
...
-You can import a previously exported API to share it across your organization or to use it at a later time. You can import an API's ZIP file from your local file system or server. 
+You can import a previously exported API to share across your organization or to use at a later time. You can import the exported API ZIP file from your local file system or server. 
 
 [NOTE]
 
-You can import only the API ZIP file that was previously exported from API Manager. You must import the API to the same business group as the one from which you exported in the source location.
+You can import only an API ZIP file that was previously exported from API Manager. You must import the API to the same business group as the one from which you exported in the source location.
 
-To create an API instance by Importing to API Manager:
+To create an API instance by importing to API Manager:
```
Comment (dhirsch1) [inline modules/ROOT/pages/import-api-task.adoc]: Suggtion: "by importing it to..."

**Review by dhirsch1 — DISMISSED** [review]
One suggestion

File: modules/ROOT/pages/import-api-task.adoc (hunk 1 above)
Comment (bjoson-mulesoft) [reply modules/ROOT/pages/import-api-task.adoc]: Done.

**Review by dhirsch1 — APPROVED** [review]

**Pull request merged**